### DISCUSSION TITLE
My IP is blocked on code.jquery.com, can't use jquery websites that uses this url to reference jquery

### DISCUSSION THREAD

**Discussion by vincejv** [jquery/jquery#5300] [Q&A]
Hello all! Good day, it seems like my IP is blocked from accessing [code.jquery.com](https://code.jquery.com) abd i couldn't properly use jquery sites that uses that uses this url to reference the jquery script... Is there any way to fix this?

**Comment by SilasRodrigues19** [comment]
Hey @vincejv 

You can use a CDN, such as cdnjs. You can link jQuery in your project in the same way, you will only change the link but the assets are the same.

Link: https://cdnjs.com/libraries/jquery

Just copy the script tag with the link and use
